packages/dev-frontend/.yalc/@liquity/chicken-bonds/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@liquity/chicken-bonds",
+  "version": "0.0.1",
+  "license": "MIT",
+  "scripts": {
+    "postinstall": "run-s generate-types compile",
```

Comment:
I don't think this should be in postinstall. If someone tries to build dev-frontend without linking a ChickenBonds repo (such as in CI), this will still run, but there are no sources to compile.